### DISCUSSION TITLE
[Help Wanted] Transform a hashmap of nodes and edges into a DiGraphMap

### DISCUSSION THREAD

**Discussion by Ben-PH** [petgraph/petgraph#691] [General]
### Summary

I have two hashmaps. One for nodes keyed by their id number, and one for edges that have "left" and "right" fields containing the id's of the nodes they are connecting. I would like to be able to transform these two maps into a `DiGraphMap`

### Motivation
I'm building a web-app. In the backend, I have a graph database (arangodb), which i query to construct these two maps. I serialize them and send the pair of maps as `json` in the response. In the app, I deserialize, but I now need to hold them as a graph. Being able to take the two graphs "as is" means overhead should be eliminated

**Comment by wangjiawen2013** [comment]
+1
